vaadin-text-field-flow-parent/vaadin-text-field-flow/src/test/java/com/vaadin/flow/component/textfield/tests/BigDecimalFieldTest.java

```diff
@@ -33,6 +34,7 @@
     @Before
     public void init() {
         field = new BigDecimalField();
+        field.setLocale(Locale.US);
```

Comment:
For some reason one test in this suite was not affected by the global setting. Configuring the field explicitly works though. Since it's only this one occurrence it should be OK.